src/benchmark/framework/experiment/setupDSL/benchmarkingBundleBuilder.ts

```diff
@@ -20,8 +21,10 @@ export type CorrespondingInputParams<T extends LLMServiceStringIdentifier> =
           : T extends "grazie"
             ? InputBenchmarkingModelParams.GrazieParams
             : T extends "lmstudio"
-              ? InputBenchmarkingModelParams.LMStudioParams
-              : never;
+              ? InputBenchmarkingModelParams.DeepSeekParams
```

Comment:
So yay, here it'd be better to put `DeepSeek` at the end, as you just did it above 

Also, here you might created a bug ❗️: now the type checks for `lmstudio` and `deepseek` are of the wrong order, so the incorrect models-params interface will be resolved.

Reply:
Oopps, here I really made a bug, my bad, sorry